src/category_theory/abelian/exact.lean

```diff
@@ -186,4 +190,22 @@ lemma epi_iff_cokernel_π_eq_zero : epi f ↔ cokernel.π f = 0 :=
 
 end
 
-end category_theory.abelian
+end abelian
+
+namespace functor
+variables {D : Type u₂} [category.{v₂} D] [abelian D]
+variables (F : C ⥤ D) {X Y Z : C} (f : X ⟶ Y) (g : Y ⟶ Z)
+
+lemma reflects_exact_of_preserves_zero_morphisms_of_faithful [preserves_zero_morphisms F]
```

Comment:
Should we have a class `reflects_exact` or something similar? I see we have similar classes, for example [category_theory.limits.reflects_colimits_of_shape](https://leanprover-community.github.io/mathlib_docs/category_theory/limits/preserves/basic.html#category_theory.limits.reflects_colimits_of_shape). This can then be an instance.